src/login/styles/_inputs.scss

```diff
@@ -122,6 +124,33 @@ input:focus {
   letter-spacing: 0.025rem;
 }
 
+.label-helpblock-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+
+  label{
+    font-size: 1rem;
+    margin-left: -2px;
+  }
+}
+
+.help-block {
+  font-size: 1rem;
+  text-transform: lowercase;
+  padding-bottom: 3px;
+}
+
+.eduid-field-error {
+  font-size: 1rem;
+  padding-left: 8px;
+}
+
+.label-required {
```

Comment:
Could this code go inside the `.label-helpblock-container` together with the styling for the label text?